### ISSUE TITLE
Handle pass-through properties in projected queries

### ISSUE THREAD

**Issue by ascott18** [IntelliTect/Coalesce#478]
In the below query, Alpha.Beta should be part of the include tree, but only Alpha is included right now.

``` c#
db.Foo
.Include(f => f.Alpha).ThenInclude(a => a.Beta)
.Select(x => new Foo() {
	// other props
	Alpha = x.Alpha
})
```

**Comment by ascott18** [comment]
Also fails to pick up `Alpha` in `db.Foo.Select(x => new Foo() { Alpha = x.Alpha })`.

**Comment by ascott18** [comment]
As I dove into this, I came to realize that I think its _good_ that the prior .Include doesn't pass through into the include tree generated for the projection. There are any number of reasons why you might want to load navigations like that, but exclude those navigations from the final mapped results of the projection. Its easy to add additional members to the include tree (add some .ThenIncludes after the .Select), but its much harder to take things away.

As for my second comment, I don't know how I came up with that. It doesn't appear to be true.
